src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,18 @@
 import React from 'react';
+import { CommentInfo } from '../CommentInfo';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+import { Comment } from '../../types/Comment';
+
+import './CommentList.scss';
+
+type Props = {
+  comments: Comment[];
+};
+
+export const CommentList = ({ comments }: Props) => (
+  <div className="commentList">
```

Comment:
what about my last comment here ? 

![image](https://github.com/mate-academy/react_static-list-of-posts/assets/30219274/236662d4-27f9-4e43-84b8-7bca83de7169)

Reply:
Ok fixed! :D